packages/api-svc/test/unit/settlement_api_svc_route.test.ts

```diff
@@ -0,0 +1,985 @@
+const mockAuthorizationClientNoAuth: IAuthorizationClient = new AuthorizationClientMock(logger, false);
+const mockAuditClient: IAuditClient = new AuditClientMock(logger);
+const mockAccountsAndBalancesAdapter: IAccountsBalancesAdapter = new AccountsBalancesAdapterMock();
+const mockConfigRepo: ISettlementConfigRepo = new SettlementConfigRepoMock();
+const mockParticipantAccountNotifier: IParticipantAccountNotifier = new ParticipantAccountNotifierMock();
+
+const server = (process.env["SETTLEMENT_SVC_URL"] || "http://localhost:3600");
+const AUTH_TOKEN = "bearer: FAKETOKEN";
+
+let mockedSettlementBatch: ISettlementBatch;
+let mockedSettlementBatchTransfers: ISettlementBatchTransfer[];
+let mockedSettlementMatrix: ISettlementMatrix;
+let mockedSettlementMatrixBatches: ISettlementMatrixBatch[];
+let mockedSettlementMatrixBatchAccount: ISettlementMatrixBatchAccount[];
+let mockedSettlementMatrixParticipantBalances: ISettlementMatrixParticipantBalance[];
+
+describe("Settlement BC api-svc route test", () => {
+    beforeAll(async () => {
+
+
+        await Service.start(
+            logger,
+            tokenHelper,
+            mockAuthorizationClientNoAuth,
+            mockAuditClient,
+            mockAccountsAndBalancesAdapter,
+            mockConfigRepo,
+            mockBatchRepo,
+            mockBatchTransferRepo,
+            mockParticipantAccountNotifier,
+            mockMatrixRequestRepo,
+            mockMessageProducer
+        );
+
+        //Prepare mocked batch data
+        mockedSettlementBatch =
+        {
+            id: "DEFAULT.USD.2023.06.19.08.30.001",
+            timestamp: Date.now(),
+            settlementModel: 'DEFAULT',
+            currencyCode: 'USD',
+            batchName: "DEFAULT.USD.2023.06.19.08.30",
+            batchSequence: 2, //100 EURO
+            state: "OPEN",
+            accounts: []
+        };
+
+        //Prepare mocked batchTransfer data
+        mockedSettlementBatchTransfers = [
+            {
+                transferId: "T001",
+                transferTimestamp: Date.now(),
+                payerFspId: "FSP-A",
+                payeeFspId: "FSP-B",
+                currencyCode: "USD",
+                amount: "10",
+                batchId: "DEFAULT.USD.2023.06.19.08.30.001",
+                batchName: "DEFAULT.USD.2023.06.19.08.30",
+                journalEntryId: "string",
+                matrixId: "SM001"
+            },
+            {
+                transferId: "T002",
+                transferTimestamp: Date.now(),
+                payerFspId: "FSP-A",
+                payeeFspId: "FSP-B",
+                currencyCode: "USD",
+                amount: "5",
+                batchId: "DEFAULT.USD.2023.06.19.08.30.001",
+                batchName: "DEFAULT.USD.2023.06.19.08.30",
+                journalEntryId: "string",
+                matrixId: "SM001"
+            }
+        ];
+
+        await mockBatchTransferRepo.storeBatchTransfer(mockedSettlementBatchTransfers[0]);
```

Comment:
Perhaps we should update `storeBatchTransfer` to support spreads. 

`storeBatchTransfer(...args: BatchTransfers[])`